### PR TITLE
6 commit added

### PR DESCRIPTION
**Pull request by mgovorovsky**
Hello, Denis! Could you please check the homework? Thank you in advance!

### CODE REVIEW

File: Homework_3.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+print("Введите значение 1:") 
+a = int(input())
+print("Введите значение 2:") 
+b = int(input())
+print("Введите значение 3:") 
+c = int(input())
+step_1 = a != 0 and b != 0 and c != 0 and "Нет нулевых значений!!!"
```
Comment (dvk-net) [inline Homework_3.py]: just a and b and c and "нет 0"

File: Homework_3.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+print(step_1)
+step_2 = 0 or a or b or c or "Введены все нули!"
```
Comment (dvk-net) [inline Homework_3.py]: What for the first 0 is here?

**Review by dvk-net — APPROVED** [review]
good

File: Homework_3.py (hunk 1 above)
Comment (mgovorovsky) [reply Homework_3.py]: corrected

File: Homework_3.py (hunk 2 above)
Comment (mgovorovsky) [reply Homework_3.py]: fixed

**Pull request merged**